src/chartkit/components/HorizontalBar.js

```diff
@@ -52,7 +55,8 @@ class HorizontalBar extends Component {
   }
 
   onMouseEnter(data, e) {
-    e.target.style.cursor = 'pointer';
+    //@NOTE Re-enable once graph is clickable
```

Comment:
V important. User Dashboard has charts with pointer for clickable currently. This breaks it.

Reply:
@ciaranschutte Was requested to remove this for now by @rosibaj

Reply:
@ciaranschutte No "easy" way to remove temporarily, so putting back.

Reply:
@ciaranschutte @nyanofthemoon Not sure on code here, but its _very_ important to have the pointer where needed, but not where not needed.  Can we work out a way for that to happen?